.github/workflows/generate-release-notes.yml

```diff
@@ -0,0 +1,57 @@
+name: "Post Release Tasks"
+# this must be triggered manually
+on: workflow_dispatch
+jobs:
+  changelog:
```

Comment:
Recommend updating to `post-release:` since it's doing more than just updating the changelog